package.json

```diff
@@ -5,9 +5,10 @@
 	],
 	"version": "0.0.1",
 	"private": true,
+	"packageManager": "npm@8.5.1",
```

Comment:
Doesn't newer turbo have this in their turbo.json file?

Reply:
Nope https://turborepo.org/blog/turbo-1-1-0#faster-package-manager-detection